.azure/pipelines/helix-matrix.yml

```diff
@@ -1,5 +1,9 @@
 # We only want to run full helix matrix on master
-pr: none
```

Comment:
revert before merge

Reply:
FYI you can kick off an aspnetcore-helix-matrix run manually from within AzDO once the branch is available in GitHub.

Reply:
Right, and its saying something that I prefer to do this and undo the merge after things go green :)